.github/workflows/pr-test.yml

```
@@ -0,0 +1,125 @@
+# Note that this workflow uses `get-diff-action` which is important.
+# It gives us an environment variable called `GIT_DIFF` which is a string
+# that is a list of file paths.
+# If you want to do what this workflow does on your laptop you can simulate
+# it like this:
+#
+#    export GIT_DIFF=`git diff --name-only main... | grep 'files/' | grep '\.html$'`
+#    node content build --no-progressbar --files="${GIT_DIFF}"
+#
+# That way, you can behave on your laptop like, this action behaves here.
+
+name: PR Test
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/checkout@v2
+        with:
+          repository: mdn/content
+          path: mdn/content
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: "12"
+
+      - name: Cache node_modules
+        uses: actions/cache@v2.1.4
+        id: cached-node_modules
+        with:
+          path: |
+            {{ github.workspace }}/mdn/content/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install all yarn packages
+        if: steps.cached-node_modules.outputs.cache-hit != 'true'
+        working-directory: ${{ github.workspace }}/mdn/content
+        run: |
+          yarn --frozen-lockfile
+
+      - uses: technote-space/get-diff-action@v4.0.5
+        id: git_diff_content
+        with:
+          PATTERNS: files/**/*.html
+          SET_ENV_NAME: GIT_DIFF_CONTENT
+
+      - name: Build changed content
+        if: ${{ env.GIT_DIFF_CONTENT }}
+        env:
+          content_root: ${{ github.workspace }}/mdn/content/files
```

Does that work?!
Normally, it's the same uppercase as we use it. I.e.
```suggestion
          CONTENT_ROOT: ${{ github.workspace }}/mdn/content/files
```
Either way, I wish we would write it out in upper case because it makes it more recognizable and easier to read.

---

If I read the docs correct that should work but let's upper case them.